bin/generate_core_lockfile.rb

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby
+
+require 'bundler'
+ENV["BUNDLE_GEMFILE"] = "/tmp/lock_generator/Gemfile"
+
+#Set up working directory
+require 'fileutils'
+FileUtils.mkdir_p("/tmp/lock_generator/bundler.d")
+FileUtils.cp("/mnt/Gemfile", "/tmp/lock_generator/")
+FileUtils.cp("/mnt/Gemfile.release.rb", "/tmp/lock_generator/bundler.d/") if File.file?("/mnt/Gemfile.release.rb")
+branch = Bundler.load.dependencies.detect { |i| i.name == "manageiq-api" }.branch
```

Comment:
Why manageiq-api?  Just curious if there was a specific reason or a random selection.

Reply:
No particular reason, just had to get the branch name from somewhere.